gulpfile.js

```diff
@@ -416,3 +418,46 @@ gulp.task('test-e2e', function() {
     'stop-webserver'
   );
 });
+
+/**
+ * Deploys production-optimized app build to the "origin/gh-branch". Every new
```

Comment:
So here is short description what it does. Honestly I like the fact that every deployment is a separate commit, so it's easy to track various revisions and get back to the last stable if needed. Also it's quite handy that not-stashed changes are deployed as well (but there will a red warning :)).